Patch for #316191 curl-config doesn't advertise support of SSL and HTTPS when
built against nss	(Lubomir Kundrak <lkundrak@redhat.com>)

--- curl-7.16.4/curl-config.in.sslconfig	2007-10-02 22:55:15.000000000 +0200
+++ curl-7.16.4/curl-config.in	2007-10-02 22:58:19.000000000 +0200
@@ -56,6 +56,8 @@
     usage 1
 fi
 
+USE_SSL="@USE_SSLEAY@@USE_GNUTLS@@USE_NSS@"
+
 while test $# -gt 0; do
     case "$1" in
     # this deals with options in the style
@@ -80,9 +82,9 @@
 
     --feature|--features)
 	if test "@USE_SSLEAY@" = "1"; then
-          echo "SSL"
           NTLM=1 # OpenSSL implies NTLM
-        elif test -n "@USE_GNUTLS@"; then
+	fi
+        if test -n "$USE_SSL"; then
           echo "SSL"
         fi
 	if test "@KRB4_ENABLED@" = "1"; then
@@ -112,13 +114,13 @@
     --protocols)
 	if test "@CURL_DISABLE_HTTP@" != "1"; then
           echo "HTTP"
-	  if test "@USE_SSLEAY@" = "1"; then
+	  if test "$USE_SSL" = "1"; then
             echo "HTTPS"
           fi
         fi
 	if test "@CURL_DISABLE_FTP@" != "1"; then
           echo "FTP"
-	  if test "@USE_SSLEAY@" = "1"; then
+	  if test "$USE_SSL" = "1"; then
             echo "FTPS"
           fi
         fi
